zulip_bots/zulip_bots/bots/file_uploader/file_uploader.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any
+
+from pathlib import Path
+
+class FileUploaderHanlder(object):
```

Comment:
Is `FileUploaderHandler` the intended name?

Reply:
Yes, inspired by `LinkShortenerHandler` :)

Reply:
Right - I was meaning whether you noticed the typo :)

Reply:
I noticed now :) Corrected